# University Admission Decision

## Task 1

In [2]:
import numpy as np
adm_data=np.genfromtxt('DS1_C4_S1_AdmissionPredict_Data_Practice.csv',delimiter=',',skip_header=1)

In [3]:
adm_data

array([[  1.  , 337.  , 118.  , ...,   9.65,   1.  ,   0.92],
       [  2.  , 324.  , 107.  , ...,   8.87,   1.  ,   0.76],
       [  3.  , 316.  , 104.  , ...,   8.  ,   1.  ,   0.72],
       ...,
       [398.  , 330.  , 116.  , ...,   9.45,   1.  ,   0.91],
       [399.  , 312.  , 103.  , ...,   8.78,   0.  ,   0.67],
       [400.  , 333.  , 117.  , ...,   9.66,   1.  ,   0.95]])

Conclusion - Imorted the dsta files in 2-D array 

## Task 2 

In [4]:
data=adm_data[:,1]

In [5]:
data

array([337., 324., 316., 322., 314., 330., 321., 308., 302., 323., 325.,
       327., 328., 307., 311., 314., 317., 319., 318., 303., 312., 325.,
       328., 334., 336., 340., 322., 298., 295., 310., 300., 327., 338.,
       340., 331., 320., 299., 300., 304., 307., 308., 316., 313., 332.,
       326., 322., 329., 339., 321., 327., 313., 312., 334., 324., 322.,
       320., 316., 298., 300., 311., 309., 307., 304., 315., 325., 325.,
       327., 316., 318., 328., 332., 336., 321., 314., 314., 329., 327.,
       301., 296., 294., 312., 340., 320., 322., 340., 319., 315., 317.,
       314., 316., 318., 299., 298., 301., 303., 304., 306., 331., 332.,
       323., 322., 312., 314., 317., 326., 316., 329., 338., 331., 304.,
       305., 321., 301., 320., 311., 310., 299., 290., 296., 327., 335.,
       334., 310., 308., 301., 300., 323., 319., 326., 333., 339., 303.,
       309., 323., 333., 314., 312., 316., 326., 318., 329., 332., 331.,
       340., 325., 320., 315., 326., 339., 311., 33

Conclusion - The GRE scores for the students is displayed

## Task 3

In [6]:
GRE = data > 320

In [7]:
filter=data[GRE]

In [8]:
filter

array([337., 324., 322., 330., 321., 323., 325., 327., 328., 325., 328.,
       334., 336., 340., 322., 327., 338., 340., 331., 332., 326., 322.,
       329., 339., 321., 327., 334., 324., 322., 325., 325., 327., 328.,
       332., 336., 321., 329., 327., 340., 322., 340., 331., 332., 323.,
       322., 326., 329., 338., 331., 321., 327., 335., 334., 323., 326.,
       333., 339., 323., 333., 326., 329., 332., 331., 340., 325., 326.,
       339., 334., 332., 321., 324., 326., 329., 322., 334., 322., 323.,
       321., 329., 327., 335., 331., 324., 324., 323., 322., 336., 340.,
       334., 325., 328., 338., 333., 331., 330., 322., 321., 324., 324.,
       324., 330., 326., 325., 329., 324., 325., 328., 324., 321., 335.,
       321., 324., 326., 331., 327., 324., 325., 327., 322., 329., 321.,
       340., 331., 336., 324., 330., 322., 323., 321., 323., 325., 328.,
       324., 327., 323., 326., 324., 327., 325., 332., 323., 324., 326.,
       325., 327., 321., 322., 334., 338., 330., 32

In [9]:
count = len(filter)
count

163

Conclusion - The total number of students who have GRE greater than 320 is 163 students.

## Task 4  

In [10]:
stu=np.where(adm_data[:,2]>110)
chance_ad=adm_data[ :,8]*100
chance_ad[stu]

array([92., 90., 84., 78., 70., 94., 95., 97., 94., 91., 90., 94., 87.,
       91., 86., 89., 78., 78., 72., 52., 55., 61., 78., 94., 96., 93.,
       72., 74., 96., 92., 94., 86., 90., 79., 74., 87., 91., 93., 94.,
       94., 85., 78., 84., 92., 96., 79., 89., 80., 90., 92., 97., 80.,
       81., 83., 96., 93., 94., 86., 81., 89., 89., 87., 85., 90., 89.,
       93., 93., 88., 90., 86., 94., 97., 97., 95., 96., 94., 93., 91.,
       76., 71., 82., 91., 88., 85., 86., 70., 76., 77., 93., 85., 90.,
       70., 83., 89., 80., 94., 93., 92., 89., 86., 90., 71., 80., 79.,
       80., 81., 80., 80., 83., 94., 93., 91., 86., 95., 96., 96., 84.,
       89., 91., 95.])

Conclusion - Students with TOFEL score greater than 110 and finding the chance admit of them and multiplying it by 100

# Task 5

In [15]:
scholor=np.where((adm_data[:,1]> 315) & (adm_data[:,2]>115) & (adm_data[:,-2]==1))

In [16]:
scholor

(array([  0,  22,  23,  24,  25,  32,  52,  70,  81,  97,  98, 107, 108,
        120, 121, 129, 138, 141, 143, 148, 151, 171, 176, 187, 193, 202,
        203, 212, 213, 214, 215, 259, 285, 286, 325, 337, 361, 372, 385,
        397, 399], dtype=int64),)

Conclusion - The students who are eligible for the scholorship as per the condition are mentioned.